{1} setUpClass (tempest.api.compute.volumes.test_attach_volume.AttachVolumeMultiAttachTest) ... SKIPPED: Neutron is not available
{1} tempest.api.compute.volumes.test_volumes_get.VolumesGetTestJSON.test_volume_create_get_delete [2.963683s] ... ok
{0} tempest.api.compute.volumes.test_volume_snapshots.VolumesSnapshotsTestJSON.test_volume_snapshot_create_get_list_delete [7.617068s] ... ok
{2} tempest.api.compute.volumes.test_volumes_list.VolumesTestJSON.test_volume_list [0.072072s] ... ok
{2} tempest.api.compute.volumes.test_volumes_list.VolumesTestJSON.test_volume_list_param_limit [0.098199s] ... ok
{2} tempest.api.compute.volumes.test_volumes_list.VolumesTestJSON.test_volume_list_param_offset_and_limit [0.139758s] ... ok
{2} tempest.api.compute.volumes.test_volumes_list.VolumesTestJSON.test_volume_list_with_detail_param_limit [0.070051s] ... ok
{2} tempest.api.compute.volumes.test_volumes_list.VolumesTestJSON.test_volume_list_with_detail_param_offset_and_limit [0.159093s] ... ok
{2} tempest.api.compute.volumes.test_volumes_list.VolumesTestJSON.test_volume_list_with_details [0.069466s] ... ok
{1} tempest.api.volume.admin.test_backends_capabilities.BackendsCapabilitiesAdminTestsJSON.test_compare_volume_stats_values [1.070244s] ... ok
{0} tempest.api.compute.volumes.test_volumes_negative.VolumesNegativeTest.test_create_volume_with_invalid_size [0.187116s] ... ok
{0} tempest.api.compute.volumes.test_volumes_negative.VolumesNegativeTest.test_create_volume_with_size_zero [0.015689s] ... ok
{0} tempest.api.compute.volumes.test_volumes_negative.VolumesNegativeTest.test_create_volume_without_passing_size [0.015539s] ... ok
{0} tempest.api.compute.volumes.test_volumes_negative.VolumesNegativeTest.test_delete_invalid_volume_id [0.047233s] ... ok
{0} tempest.api.compute.volumes.test_volumes_negative.VolumesNegativeTest.test_delete_volume_without_passing_volume_id [0.011477s] ... ok
{0} tempest.api.compute.volumes.test_volumes_negative.VolumesNegativeTest.test_get_volume_without_passing_volume_id [0.013886s] ... ok
{0} tempest.api.compute.volumes.test_volumes_negative.VolumesNegativeTest.test_volume_delete_nonexistent_volume_id [0.045544s] ... ok
{0} tempest.api.compute.volumes.test_volumes_negative.VolumesNegativeTest.test_volume_get_nonexistent_volume_id [0.051923s] ... ok
{1} tempest.api.volume.admin.test_backends_capabilities.BackendsCapabilitiesAdminTestsJSON.test_get_capabilities_backend [0.535962s] ... ok
{0} setUpClass (tempest.api.volume.admin.test_encrypted_volumes_extend.EncryptedVolumesExtendAttachedTest) ... SKIPPED: EncryptedVolumesExtendAttachedTest skipped as Neutron is not available
{2} tempest.api.volume.admin.test_groups.GroupsTest.test_group_create_show_list_delete [8.444622s] ... ok
{1} tempest.api.volume.admin.test_groups.GroupsV314Test.test_create_group_from_group [11.982543s] ... ok
{0} tempest.api.volume.admin.test_group_snapshots.GroupSnapshotsTest.test_create_group_from_group_snapshot [14.358367s] ... ok
{1} tempest.api.volume.admin.test_qos.QosSpecsTestJSON.test_associate_disassociate_qos [0.771762s] ... ok
{1} tempest.api.volume.admin.test_qos.QosSpecsTestJSON.test_create_delete_qos_with_back_end_consumer [0.158767s] ... ok
{1} tempest.api.volume.admin.test_qos.QosSpecsTestJSON.test_create_delete_qos_with_both_consumer [0.168797s] ... ok
{1} tempest.api.volume.admin.test_qos.QosSpecsTestJSON.test_create_delete_qos_with_front_end_consumer [0.172745s] ... ok
{1} tempest.api.volume.admin.test_qos.QosSpecsTestJSON.test_get_qos [0.025133s] ... ok
{1} tempest.api.volume.admin.test_qos.QosSpecsTestJSON.test_list_qos [0.026691s] ... ok
{1} tempest.api.volume.admin.test_qos.QosSpecsTestJSON.test_set_unset_qos_key [0.156394s] ... ok
{2} tempest.api.volume.admin.test_groups.GroupsTest.test_group_update [10.892514s] ... ok
{2} setUpClass (tempest.api.volume.admin.test_snapshot_manage.SnapshotManageAdminTest) ... SKIPPED: SnapshotManageAdminTest skipped as Neutron is not available
{2} tempest.api.volume.admin.test_user_messages.UserMessagesTest.test_delete_message [2.177450s] ... ok
{2} tempest.api.volume.admin.test_user_messages.UserMessagesTest.test_list_show_messages [2.062127s] ... ok
{1} tempest.api.volume.admin.test_snapshots_actions.SnapshotsActionsTest.test_reset_snapshot_status [0.638357s] ... ok
{0} tempest.api.volume.admin.test_group_snapshots.GroupSnapshotsTest.test_delete_group_snapshots_following_updated_volumes [17.801243s] ... ok
{1} tempest.api.volume.admin.test_snapshots_actions.SnapshotsActionsTest.test_snapshot_force_delete_when_snapshot_is_creating [3.623311s] ... ok
{2} tempest.api.volume.admin.test_volume_hosts.VolumeHostsAdminTestsJSON.test_list_hosts [0.222640s] ... ok
{2} tempest.api.volume.admin.test_volume_hosts.VolumeHostsAdminTestsJSON.test_show_host [0.088356s] ... ok
{1} tempest.api.volume.admin.test_snapshots_actions.SnapshotsActionsTest.test_snapshot_force_delete_when_snapshot_is_deleting [3.627964s] ... ok
{2} tempest.api.volume.admin.test_volume_quota_classes.VolumeQuotaClassesTest.test_show_default_quota [0.476618s] ... ok
{0} tempest.api.volume.admin.test_group_snapshots.GroupSnapshotsTest.test_group_snapshot_create_show_list_delete [9.160759s] ... ok
{1} tempest.api.volume.admin.test_snapshots_actions.SnapshotsActionsTest.test_snapshot_force_delete_when_snapshot_is_error [3.617516s] ... ok
{2} tempest.api.volume.admin.test_volume_quota_classes.VolumeQuotaClassesTest.test_update_default_quota [2.141358s] ... ok
{1} tempest.api.volume.admin.test_snapshots_actions.SnapshotsActionsTest.test_snapshot_force_delete_when_snapshot_is_error_deleting [3.612486s] ... ok
{1} tempest.api.volume.admin.test_snapshots_actions.SnapshotsActionsTest.test_update_snapshot_status [0.236183s] ... ok
{2} tempest.api.volume.admin.test_volume_quotas.VolumeQuotasAdminTestJSON.test_delete_quota [1.026182s] ... ok
{2} tempest.api.volume.admin.test_volume_quotas.VolumeQuotasAdminTestJSON.test_list_default_quotas [0.177391s] ... ok
{2} tempest.api.volume.admin.test_volume_quotas.VolumeQuotasAdminTestJSON.test_list_quotas [0.186163s] ... ok
{1} tempest.api.volume.admin.test_volume_pools.VolumePoolsAdminTestsJSON.test_get_pools_without_details [0.697498s] ... ok
{2} tempest.api.volume.admin.test_volume_quotas.VolumeQuotasAdminTestJSON.test_quota_usage [4.381462s] ... ok
{0} tempest.api.volume.admin.test_group_snapshots.GroupSnapshotsV319Test.test_reset_group_snapshot_status [9.838468s] ... ok
{0} tempest.api.volume.admin.test_group_type_specs.GroupTypeSpecsTest.test_group_type_specs_create_show_update_list_delete [0.547143s] ... ok
{2} tempest.api.volume.admin.test_volume_quotas.VolumeQuotasAdminTestJSON.test_quota_usage_after_volume_transfer [7.310353s] ... ok
{2} tempest.api.volume.admin.test_volume_quotas.VolumeQuotasAdminTestJSON.test_show_quota_usage [0.207431s] ... ok
{1} tempest.api.volume.admin.test_volume_retype.VolumeRetypeWithoutMigrationTest.test_available_volume_retype [5.373727s] ... ok
{2} tempest.api.volume.admin.test_volume_quotas.VolumeQuotasAdminTestJSON.test_update_all_quota_resources_for_tenant [0.738967s] ... ok
{0} tempest.api.volume.admin.test_group_types.GroupTypesTest.test_group_type_create_list_update_show_delete [0.423972s] ... ok
{0} tempest.api.volume.admin.test_group_types.GroupTypesTest.test_group_type_list_by_optional_params [0.265030s] ... ok
{1} tempest.api.volume.admin.test_volume_type_access.VolumeTypesAccessTest.test_volume_type_access_list [0.461838s] ... ok
{1} setUpClass (tempest.api.volume.admin.test_volumes_actions.VolumesActionsTest) ... SKIPPED: VolumesActionsTest skipped as Neutron is not available
{2} tempest.api.volume.admin.test_volume_quotas_negative.VolumeQuotasNegativeTestJSON.test_quota_volume_gigabytes [0.882833s] ... ok
{2} tempest.api.volume.admin.test_volume_quotas_negative.VolumeQuotasNegativeTestJSON.test_quota_volumes [0.559519s] ... ok
{2} tempest.api.volume.admin.test_volume_quotas_negative.VolumeQuotasNegativeTestJSON.test_volume_extend_gigabytes_quota_deviation [0.505985s] ... ok
{1} setUpClass (tempest.api.volume.test_image_metadata.VolumesImageMetadata) [0.000000s] ... FAILED

Captured traceback:
~~~~~~~~~~~~~~~~~~~
    Traceback (most recent call last):

      File "/usr/local/lib/python3.10/dist-packages/tempest/test.py", line 185, in setUpClass
    raise value.with_traceback(trace)

      File "/usr/local/lib/python3.10/dist-packages/tempest/test.py", line 178, in setUpClass
    cls.resource_setup()

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/test_image_metadata.py", line 40, in resource_setup
    cls.volume = cls.create_volume(imageRef=CONF.compute.image_ref)

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/base.py", line 118, in create_volume
    image = self.images_client.show_image(kwargs['imageRef'])

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/services/image/v2/images_client.py", line 119, in show_image
    resp, body = self.get(url)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 342, in get
    return self.request('GET', url, extra_headers, headers,

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 762, in request
    self._error_checker(resp, resp_body)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 872, in _error_checker
    raise exceptions.NotFound(resp_body, resp=resp)

    tempest.lib.exceptions.NotFound: Object not found
Details: {'message': 'No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\n\n\n', 'code': '404 Not Found', 'title': 'Not Found'}

{0} tempest.api.volume.admin.test_groups.GroupsV320Test.test_reset_group_status [3.214109s] ... ok
{0} setUpClass (tempest.api.volume.admin.test_multi_backend.VolumeMultiBackendTest) ... SKIPPED: Cinder multi-backend feature disabled
{0} setUpClass (tempest.api.volume.admin.test_volume_manage.VolumeManageAdminTest) ... SKIPPED: Manage volume tests are disabled
{0} setUpClass (tempest.api.volume.admin.test_volume_retype.VolumeRetypeWithMigrationTest) ... SKIPPED: Cinder multi-backend feature disabled
{2} tempest.api.volume.admin.test_volume_services.VolumesServicesTestJSON.test_get_service_by_host_name [0.027912s] ... ok
{0} tempest.api.volume.admin.test_volume_types.VolumeTypesTest.test_volume_type_create_get_delete [0.320422s] ... ok
{2} tempest.api.volume.admin.test_volume_services.VolumesServicesTestJSON.test_get_service_by_service_and_host_name [0.029219s] ... ok
{2} tempest.api.volume.admin.test_volume_services.VolumesServicesTestJSON.test_get_service_by_service_binary_name [0.024358s] ... ok
{0} tempest.api.volume.admin.test_volume_types.VolumeTypesTest.test_volume_type_encryption_create_get_update_delete [0.468540s] ... ok
{0} tempest.api.volume.admin.test_volume_types.VolumeTypesTest.test_volume_type_list [0.037399s] ... ok
{0} tempest.api.volume.admin.test_volume_types.VolumeTypesTest.test_volume_type_update [0.497356s] ... ok
{1} tempest.api.volume.test_snapshot_metadata.SnapshotMetadataTestJSON.test_crud_snapshot_metadata [0.300866s] ... ok
{1} tempest.api.volume.test_snapshot_metadata.SnapshotMetadataTestJSON.test_update_show_snapshot_metadata_item [0.255719s] ... ok
{2} tempest.api.volume.admin.test_volume_services.VolumesServicesTestJSON.test_get_service_by_volume_host_name [4.004017s] ... ok
{2} tempest.api.volume.admin.test_volume_services.VolumesServicesTestJSON.test_list_services [0.027479s] ... ok
{0} tempest.api.volume.admin.test_volume_types_negative.VolumeTypesNegativeTest.test_create_volume_type_encryption_nonexistent_type_id [0.206209s] ... ok
{0} tempest.api.volume.admin.test_volume_types_negative.VolumeTypesNegativeTest.test_create_with_empty_name [0.017114s] ... ok
{0} tempest.api.volume.admin.test_volume_types_negative.VolumeTypesNegativeTest.test_create_with_repeated_name [0.155538s] ... ok
{0} tempest.api.volume.admin.test_volume_types_negative.VolumeTypesNegativeTest.test_delete_nonexistent_type_id [0.021104s] ... ok
{0} tempest.api.volume.admin.test_volume_types_negative.VolumeTypesNegativeTest.test_get_nonexistent_type_id [0.020451s] ... ok
{1} tempest.api.volume.test_versions.VersionsTest.test_list_versions [0.014033s] ... ok
{1} tempest.api.volume.test_versions.VersionsTest.test_show_version [0.169416s] ... ok
{2} tempest.api.volume.admin.test_volume_services_negative.VolumeServicesNegativeTest.test_disable_log_reason_with_no_reason [0.019237s] ... ok
{2} tempest.api.volume.admin.test_volume_services_negative.VolumeServicesNegativeTest.test_disable_service_with_invalid_binary [0.020523s] ... ok
{2} tempest.api.volume.admin.test_volume_services_negative.VolumeServicesNegativeTest.test_enable_service_with_invalid_host [0.024070s] ... ok
{2} tempest.api.volume.admin.test_volume_services_negative.VolumeServicesNegativeTest.test_freeze_host_with_invalid_host [0.027198s] ... ok
{2} tempest.api.volume.admin.test_volume_services_negative.VolumeServicesNegativeTest.test_thaw_host_with_invalid_host [0.021175s] ... ok
{1} tempest.api.volume.test_volume_transfers.VolumesTransfersV357Test.test_create_get_list_accept_volume_transfer [5.971123s] ... ok
{2} tempest.api.volume.admin.test_volume_snapshot_quotas_negative.VolumeSnapshotQuotasNegativeTestJSON.test_quota_volume_gigabytes_snapshots [0.984701s] ... ok
{2} tempest.api.volume.admin.test_volume_snapshot_quotas_negative.VolumeSnapshotQuotasNegativeTestJSON.test_quota_volume_snapshots [0.150855s] ... ok
{0} tempest.api.volume.admin.test_volumes_list.VolumesListAdminTestJSON.test_volume_list_param_tenant [2.818728s] ... ok
{1} tempest.api.volume.test_volume_transfers.VolumesTransfersV357Test.test_create_list_delete_volume_transfer [4.006854s] ... ok
{0} tempest.api.volume.test_availability_zone.AvailabilityZoneTestJSON.test_get_availability_zone_list [0.127095s] ... ok
{2} tempest.api.volume.admin.test_volume_types_extra_specs.VolumeTypesExtraSpecsTest.test_volume_type_extra_spec_create_get_delete [0.174558s] ... ok
{2} tempest.api.volume.admin.test_volume_types_extra_specs.VolumeTypesExtraSpecsTest.test_volume_type_extra_specs_list [0.091134s] ... ok
{2} tempest.api.volume.admin.test_volume_types_extra_specs.VolumeTypesExtraSpecsTest.test_volume_type_extra_specs_update [0.238725s] ... ok
{1} tempest.api.volume.test_volumes_clone_negative.VolumesCloneNegativeTest.test_create_from_volume_decreasing_size [4.056604s] ... ok
{1} setUpClass (tempest.api.volume.test_volumes_extend.VolumesExtendAttachedTest) ... SKIPPED: VolumesExtendAttachedTest skipped as Neutron is not available
{0} tempest.api.volume.test_extensions.ExtensionsTestJSON.test_list_extensions [0.168636s] ... ok
{1} tempest.api.volume.test_volumes_extend.VolumesExtendTest.test_volume_extend [0.178776s] ... FAILED

Captured traceback:
~~~~~~~~~~~~~~~~~~~
    Traceback (most recent call last):

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/test_volumes_extend.py", line 36, in test_volume_extend
    volume = self.create_volume(imageRef=self.image_ref)

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/base.py", line 118, in create_volume
    image = self.images_client.show_image(kwargs['imageRef'])

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/services/image/v2/images_client.py", line 119, in show_image
    resp, body = self.get(url)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 342, in get
    return self.request('GET', url, extra_headers, headers,

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 762, in request
    self._error_checker(resp, resp_body)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 872, in _error_checker
    raise exceptions.NotFound(resp_body, resp=resp)

    tempest.lib.exceptions.NotFound: Object not found
Details: {'message': 'No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\n\n\n', 'code': '404 Not Found', 'title': 'Not Found'}


Captured pythonlogging:
~~~~~~~~~~~~~~~~~~~~~~~
    2025-03-10 15:20:32,145 141060 INFO     [tempest.lib.common.rest_client] Request (VolumesExtendTest:test_volume_extend): 404 GET http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222 0.176s
2025-03-10 15:20:32,145 141060 DEBUG    [tempest.lib.common.rest_client] Request - Headers: {'Content-Type': 'application/json', 'Accept': 'application/json', 'X-Auth-Token': '<omitted>'}
        Body: None
    Response - Headers: {'content-length': '139', 'content-type': 'application/json', 'x-openstack-request-id': 'req-584188ce-b83c-4c1e-9657-e80883f60245', 'date': 'Mon, 10 Mar 2025 15:20:32 GMT', 'connection': 'close', 'status': '404', 'content-location': 'http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222'}
        Body: b'{"message": "No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\\n\\n\\n", "code": "404 Not Found", "title": "Not Found"}'

{2} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_create_invalid_body [0.018314s] ... ok
{2} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_create_none_body [0.014558s] ... ok
{2} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_create_nonexistent_type_id [0.041177s] ... ok
{2} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_delete_nonexistent_volume_type_id [0.021056s] ... ok
{2} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_get_nonexistent_extra_spec_name [0.026473s] ... ok
{2} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_get_nonexistent_volume_type_id [0.021895s] ... ok
{2} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_list_nonexistent_volume_type_id [0.021851s] ... ok
{2} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_update_multiple_extra_spec [0.018210s] ... ok
{2} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_update_no_body [0.017007s] ... ok
{2} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_update_none_extra_spec_id [0.038528s] ... ok
{2} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_update_nonexistent_extra_spec_id [0.037897s] ... ok
{2} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_update_nonexistent_type_id [0.037538s] ... ok
{0} tempest.api.volume.test_volume_absolute_limits.AbsoluteLimitsTests.test_get_volume_absolute_limits [0.220349s] ... ok
{1} tempest.api.volume.test_volumes_extend.VolumesExtendTest.test_volume_extend_when_volume_has_snapshot [8.636824s] ... ok
{2} tempest.api.volume.test_volume_delete_cascade.VolumesDeleteCascade.test_volume_delete_cascade [8.715744s] ... ok
{1} tempest.api.volume.test_volumes_get.VolumesGetTest.test_volume_create_get_update_delete [7.837054s] ... ok
{0} tempest.api.volume.test_volume_transfers.VolumesTransfersV355Test.test_create_get_list_accept_volume_transfer [5.802601s] ... ok
{2} tempest.api.volume.test_volume_delete_cascade.VolumesDeleteCascade.test_volume_from_snapshot_cascade_delete [10.530892s] ... ok
{0} tempest.api.volume.test_volume_transfers.VolumesTransfersV355Test.test_create_list_delete_volume_transfer [4.353080s] ... ok
{0} tempest.api.volume.test_volumes_clone.VolumesCloneTest.test_create_from_bootable_volume [0.187077s] ... FAILED

Captured traceback:
~~~~~~~~~~~~~~~~~~~
    Traceback (most recent call last):

      File "/usr/local/lib/python3.10/dist-packages/tempest/common/utils/__init__.py", line 65, in wrapper
    return f(*func_args, **func_kwargs)

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/test_volumes_clone.py", line 67, in test_create_from_bootable_volume
    src_vol = self.create_volume(imageRef=img_uuid)

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/base.py", line 118, in create_volume
    image = self.images_client.show_image(kwargs['imageRef'])

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/services/image/v2/images_client.py", line 119, in show_image
    resp, body = self.get(url)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 342, in get
    return self.request('GET', url, extra_headers, headers,

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 762, in request
    self._error_checker(resp, resp_body)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 872, in _error_checker
    raise exceptions.NotFound(resp_body, resp=resp)

    tempest.lib.exceptions.NotFound: Object not found
Details: {'message': 'No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\n\n\n', 'code': '404 Not Found', 'title': 'Not Found'}


Captured pythonlogging:
~~~~~~~~~~~~~~~~~~~~~~~
    2025-03-10 15:20:59,827 141058 INFO     [tempest.lib.common.rest_client] Request (VolumesCloneTest:test_create_from_bootable_volume): 404 GET http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222 0.183s
2025-03-10 15:20:59,827 141058 DEBUG    [tempest.lib.common.rest_client] Request - Headers: {'Content-Type': 'application/json', 'Accept': 'application/json', 'X-Auth-Token': '<omitted>'}
        Body: None
    Response - Headers: {'content-length': '139', 'content-type': 'application/json', 'x-openstack-request-id': 'req-2508689d-93e1-4e7d-a32d-f49b7836dc41', 'date': 'Mon, 10 Mar 2025 15:20:59 GMT', 'connection': 'close', 'status': '404', 'content-location': 'http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222'}
        Body: b'{"message": "No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\\n\\n\\n", "code": "404 Not Found", "title": "Not Found"}'

{2} tempest.api.volume.test_volume_metadata.VolumesMetadataTest.test_crud_volume_metadata [0.348974s] ... ok
{2} tempest.api.volume.test_volume_metadata.VolumesMetadataTest.test_update_show_volume_metadata_item [0.233331s] ... ok
{1} tempest.api.volume.test_volumes_get.VolumesGetTest.test_volume_create_get_update_delete_as_clone [12.597416s] ... ok
{1} tempest.api.volume.test_volumes_get.VolumesGetTest.test_volume_create_get_update_delete_from_image [0.021489s] ... FAILED

Captured traceback:
~~~~~~~~~~~~~~~~~~~
    Traceback (most recent call last):

      File "/usr/local/lib/python3.10/dist-packages/tempest/common/utils/__init__.py", line 65, in wrapper
    return f(*func_args, **func_kwargs)

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/test_volumes_get.py", line 132, in test_volume_create_get_update_delete_from_image
    image = self.images_client.show_image(CONF.compute.image_ref)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/services/image/v2/images_client.py", line 119, in show_image
    resp, body = self.get(url)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 342, in get
    return self.request('GET', url, extra_headers, headers,

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 762, in request
    self._error_checker(resp, resp_body)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 872, in _error_checker
    raise exceptions.NotFound(resp_body, resp=resp)

    tempest.lib.exceptions.NotFound: Object not found
Details: {'message': 'No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\n\n\n', 'code': '404 Not Found', 'title': 'Not Found'}


Captured pythonlogging:
~~~~~~~~~~~~~~~~~~~~~~~
    2025-03-10 15:21:04,141 141060 INFO     [tempest.lib.common.rest_client] Request (VolumesGetTest:test_volume_create_get_update_delete_from_image): 404 GET http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222 0.019s
2025-03-10 15:21:04,142 141060 DEBUG    [tempest.lib.common.rest_client] Request - Headers: {'Content-Type': 'application/json', 'Accept': 'application/json', 'X-Auth-Token': '<omitted>'}
        Body: None
    Response - Headers: {'content-length': '139', 'content-type': 'application/json', 'x-openstack-request-id': 'req-e420ab54-f4fd-4742-ab92-5943c9a82b59', 'date': 'Mon, 10 Mar 2025 15:21:04 GMT', 'connection': 'close', 'status': '404', 'content-location': 'http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222'}
        Body: b'{"message": "No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\\n\\n\\n", "code": "404 Not Found", "title": "Not Found"}'

{0} tempest.api.volume.test_volumes_clone.VolumesCloneTest.test_create_from_volume [9.775431s] ... ok
{2} tempest.api.volume.test_volume_transfers.VolumesTransfersTest.test_create_get_list_accept_volume_transfer [5.936950s] ... ok
{1} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_limit [0.027321s] ... ok
{1} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_limit_equals_infinite [0.058159s] ... ok
{1} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_limit_equals_zero [0.027079s] ... ok
{1} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_marker [0.067713s] ... ok
{1} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_offset [0.028639s] ... ok
{1} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_sort_created_at_asc [0.030919s] ... ok
{1} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_sort_created_at_desc [0.029960s] ... ok
{1} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_sort_id_asc [0.030612s] ... ok
{1} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_sort_id_desc [0.030505s] ... ok
{1} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_sort_name_asc [0.031200s] ... ok
{1} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_sort_name_desc [0.030931s] ... ok
{1} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshots_list_details_with_params [0.093030s] ... ok
{1} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshots_list_with_params [0.076931s] ... ok
{2} tempest.api.volume.test_volume_transfers.VolumesTransfersTest.test_create_list_delete_volume_transfer [4.067406s] ... ok
{2} setUpClass (tempest.api.volume.test_volumes_actions.VolumesActionsTest) ... SKIPPED: VolumesActionsTest skipped as Neutron is not available
{0} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list [0.036034s] ... ok
{0} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_by_name [0.036644s] ... ok
{0} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_details_by_name [0.057011s] ... ok
{0} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_details_pagination [0.246244s] ... ok
{0} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_details_with_multiple_params [0.120963s] ... ok
{0} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_pagination [0.158890s] ... ok
{0} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_param_display_name_and_status [0.035095s] ... ok
{0} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_with_detail_param_display_name_and_status [0.058217s] ... ok
{0} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_with_detail_param_marker [0.063223s] ... ok
{0} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_with_detail_param_metadata [0.061888s] ... ok
{0} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_with_details [0.061143s] ... ok
{0} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_with_param_metadata [0.042487s] ... ok
{0} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volumes_list_by_availability_zone [0.080003s] ... ok
{0} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volumes_list_by_bootable [0.077974s] ... ok
{0} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volumes_list_by_status [0.076650s] ... ok
{0} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volumes_list_details_by_availability_zone [0.063004s] ... ok
{0} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volumes_list_details_by_bootable [0.060044s] ... ok
{0} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volumes_list_details_by_status [0.064583s] ... ok
{2} tempest.api.volume.test_volumes_get.VolumesSummaryTest.test_show_volume_summary [0.206184s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_create_volume_from_deactivated_image [0.598154s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_create_volume_from_image_with_decreasing_size [0.410235s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_create_volume_with_invalid_size [0.018510s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_create_volume_with_nonexistent_snapshot_id [0.029882s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_create_volume_with_nonexistent_source_volid [0.039175s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_create_volume_with_nonexistent_volume_type [0.035189s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_create_volume_with_size_negative [0.017780s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_create_volume_with_size_zero [0.017719s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_create_volume_without_passing_size [0.016389s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_delete_invalid_volume_id [0.027701s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_delete_volume_without_passing_volume_id [0.010988s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_detach_volumes_with_invalid_volume_id [0.031710s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_get_invalid_volume_id [0.029542s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_get_volume_without_passing_volume_id [0.011800s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_list_volumes_detail_with_invalid_status [0.052404s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_list_volumes_detail_with_nonexistent_name [0.049440s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_list_volumes_with_invalid_status [0.031613s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_list_volumes_with_nonexistent_name [0.029446s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_reserve_volume_with_negative_volume_status [0.108798s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_reserve_volume_with_nonexistent_volume_id [0.022741s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_unreserve_volume_with_nonexistent_volume_id [0.021357s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_update_volume_with_empty_volume_id [0.011439s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_update_volume_with_invalid_volume_id [0.028999s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_update_volume_with_nonexistent_volume_id [0.028463s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_volume_delete_nonexistent_volume_id [0.027488s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_volume_extend_with_None_size [0.016051s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_volume_extend_with_non_number_size [0.016027s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_volume_extend_with_nonexistent_volume_id [0.029080s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_volume_extend_with_size_smaller_than_original_size [0.015766s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_volume_extend_without_passing_volume_id [0.030342s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_volume_get_nonexistent_volume_id [0.026208s] ... ok
{2} setUpClass (tempest.api.volume.test_volumes_snapshots.VolumesSnapshotTestJSON) ... SKIPPED: VolumesSnapshotTestJSON skipped as Neutron is not available
{2} tempest.api.volume.test_volumes_snapshots_negative.VolumesSnapshotNegativeTestJSON.test_create_snapshot_with_nonexistent_volume_id [0.171191s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_negative.VolumesSnapshotNegativeTestJSON.test_create_snapshot_without_passing_volume_id [0.025053s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_negative.VolumesSnapshotNegativeTestJSON.test_list_snapshot_invalid_param_limit [0.011860s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_negative.VolumesSnapshotNegativeTestJSON.test_list_snapshots_invalid_param_marker [0.020392s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_negative.VolumesSnapshotNegativeTestJSON.test_list_snapshots_invalid_param_sort [0.013858s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_negative.VolumesSnapshotNegativeTestJSON.test_volume_from_snapshot_decreasing_size [7.289822s] ... ok

==============================
Failed 4 tests - output below:
==============================

setUpClass (tempest.api.volume.test_image_metadata.VolumesImageMetadata)
------------------------------------------------------------------------

Captured traceback:
~~~~~~~~~~~~~~~~~~~
    Traceback (most recent call last):

      File "/usr/local/lib/python3.10/dist-packages/tempest/test.py", line 185, in setUpClass
    raise value.with_traceback(trace)

      File "/usr/local/lib/python3.10/dist-packages/tempest/test.py", line 178, in setUpClass
    cls.resource_setup()

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/test_image_metadata.py", line 40, in resource_setup
    cls.volume = cls.create_volume(imageRef=CONF.compute.image_ref)

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/base.py", line 118, in create_volume
    image = self.images_client.show_image(kwargs['imageRef'])

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/services/image/v2/images_client.py", line 119, in show_image
    resp, body = self.get(url)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 342, in get
    return self.request('GET', url, extra_headers, headers,

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 762, in request
    self._error_checker(resp, resp_body)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 872, in _error_checker
    raise exceptions.NotFound(resp_body, resp=resp)

    tempest.lib.exceptions.NotFound: Object not found
Details: {'message': 'No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\n\n\n', 'code': '404 Not Found', 'title': 'Not Found'}


tempest.api.volume.test_volumes_extend.VolumesExtendTest.test_volume_extend[id-9a36df71-a257-43a5-9555-dc7c88e66e0e]
--------------------------------------------------------------------------------------------------------------------

Captured traceback:
~~~~~~~~~~~~~~~~~~~
    Traceback (most recent call last):

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/test_volumes_extend.py", line 36, in test_volume_extend
    volume = self.create_volume(imageRef=self.image_ref)

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/base.py", line 118, in create_volume
    image = self.images_client.show_image(kwargs['imageRef'])

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/services/image/v2/images_client.py", line 119, in show_image
    resp, body = self.get(url)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 342, in get
    return self.request('GET', url, extra_headers, headers,

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 762, in request
    self._error_checker(resp, resp_body)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 872, in _error_checker
    raise exceptions.NotFound(resp_body, resp=resp)

    tempest.lib.exceptions.NotFound: Object not found
Details: {'message': 'No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\n\n\n', 'code': '404 Not Found', 'title': 'Not Found'}


Captured pythonlogging:
~~~~~~~~~~~~~~~~~~~~~~~
    2025-03-10 15:20:32,145 141060 INFO     [tempest.lib.common.rest_client] Request (VolumesExtendTest:test_volume_extend): 404 GET http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222 0.176s
2025-03-10 15:20:32,145 141060 DEBUG    [tempest.lib.common.rest_client] Request - Headers: {'Content-Type': 'application/json', 'Accept': 'application/json', 'X-Auth-Token': '<omitted>'}
        Body: None
    Response - Headers: {'content-length': '139', 'content-type': 'application/json', 'x-openstack-request-id': 'req-584188ce-b83c-4c1e-9657-e80883f60245', 'date': 'Mon, 10 Mar 2025 15:20:32 GMT', 'connection': 'close', 'status': '404', 'content-location': 'http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222'}
        Body: b'{"message": "No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\\n\\n\\n", "code": "404 Not Found", "title": "Not Found"}'


tempest.api.volume.test_volumes_clone.VolumesCloneTest.test_create_from_bootable_volume[id-cbbcd7c6-5a6c-481a-97ac-ca55ab715d16,image]
--------------------------------------------------------------------------------------------------------------------------------------

Captured traceback:
~~~~~~~~~~~~~~~~~~~
    Traceback (most recent call last):

      File "/usr/local/lib/python3.10/dist-packages/tempest/common/utils/__init__.py", line 65, in wrapper
    return f(*func_args, **func_kwargs)

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/test_volumes_clone.py", line 67, in test_create_from_bootable_volume
    src_vol = self.create_volume(imageRef=img_uuid)

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/base.py", line 118, in create_volume
    image = self.images_client.show_image(kwargs['imageRef'])

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/services/image/v2/images_client.py", line 119, in show_image
    resp, body = self.get(url)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 342, in get
    return self.request('GET', url, extra_headers, headers,

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 762, in request
    self._error_checker(resp, resp_body)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 872, in _error_checker
    raise exceptions.NotFound(resp_body, resp=resp)

    tempest.lib.exceptions.NotFound: Object not found
Details: {'message': 'No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\n\n\n', 'code': '404 Not Found', 'title': 'Not Found'}


Captured pythonlogging:
~~~~~~~~~~~~~~~~~~~~~~~
    2025-03-10 15:20:59,827 141058 INFO     [tempest.lib.common.rest_client] Request (VolumesCloneTest:test_create_from_bootable_volume): 404 GET http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222 0.183s
2025-03-10 15:20:59,827 141058 DEBUG    [tempest.lib.common.rest_client] Request - Headers: {'Content-Type': 'application/json', 'Accept': 'application/json', 'X-Auth-Token': '<omitted>'}
        Body: None
    Response - Headers: {'content-length': '139', 'content-type': 'application/json', 'x-openstack-request-id': 'req-2508689d-93e1-4e7d-a32d-f49b7836dc41', 'date': 'Mon, 10 Mar 2025 15:20:59 GMT', 'connection': 'close', 'status': '404', 'content-location': 'http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222'}
        Body: b'{"message": "No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\\n\\n\\n", "code": "404 Not Found", "title": "Not Found"}'


tempest.api.volume.test_volumes_get.VolumesGetTest.test_volume_create_get_update_delete_from_image[id-54a01030-c7fc-447c-86ee-c1182beae638,image,smoke]
-------------------------------------------------------------------------------------------------------------------------------------------------------

Captured traceback:
~~~~~~~~~~~~~~~~~~~
    Traceback (most recent call last):

      File "/usr/local/lib/python3.10/dist-packages/tempest/common/utils/__init__.py", line 65, in wrapper
    return f(*func_args, **func_kwargs)

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/test_volumes_get.py", line 132, in test_volume_create_get_update_delete_from_image
    image = self.images_client.show_image(CONF.compute.image_ref)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/services/image/v2/images_client.py", line 119, in show_image
    resp, body = self.get(url)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 342, in get
    return self.request('GET', url, extra_headers, headers,

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 762, in request
    self._error_checker(resp, resp_body)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 872, in _error_checker
    raise exceptions.NotFound(resp_body, resp=resp)

    tempest.lib.exceptions.NotFound: Object not found
Details: {'message': 'No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\n\n\n', 'code': '404 Not Found', 'title': 'Not Found'}


Captured pythonlogging:
~~~~~~~~~~~~~~~~~~~~~~~
    2025-03-10 15:21:04,141 141060 INFO     [tempest.lib.common.rest_client] Request (VolumesGetTest:test_volume_create_get_update_delete_from_image): 404 GET http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222 0.019s
2025-03-10 15:21:04,142 141060 DEBUG    [tempest.lib.common.rest_client] Request - Headers: {'Content-Type': 'application/json', 'Accept': 'application/json', 'X-Auth-Token': '<omitted>'}
        Body: None
    Response - Headers: {'content-length': '139', 'content-type': 'application/json', 'x-openstack-request-id': 'req-e420ab54-f4fd-4742-ab92-5943c9a82b59', 'date': 'Mon, 10 Mar 2025 15:21:04 GMT', 'connection': 'close', 'status': '404', 'content-location': 'http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222'}
        Body: b'{"message": "No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\\n\\n\\n", "code": "404 Not Found", "title": "Not Found"}'



======
Totals
======
Ran: 203 tests in 208.0042 sec.
 - Passed: 189
 - Skipped: 10
 - Expected Fail: 0
 - Unexpected Success: 0
 - Failed: 4
Sum of execute time for each test: 263.4249 sec.

==============
Worker Balance
==============
 - Worker 0 (56 tests) => 0:03:04.217863
 - Worker 1 (49 tests) => 0:03:02.242704
 - Worker 2 (98 tests) => 0:03:17.377995
